crates/swc/tests/fixture/issues-7xxx/7315/1/output/index.js

```diff
@@ -0,0 +1,14 @@
+// input.ts
+"use client";
```

Comment:
I believe that `"use strict"` is also necessary.

Reply:
Does it have any effect? Hmm

Reply:
AFAIK, The only first directive has an effect.

Reply:
continuous directives will be recognized in Babel.

https://github.com/babel/babel/blob/980b63c829de369ace21461d3268d04bb80fc380/packages/babel-parser/test/fixtures/estree/directives/program-order/input.js

Reply:
And all valid directive will be parsed as directive before first `non directive`.

https://github.com/babel/babel/blob/980b63c829de369ace21461d3268d04bb80fc380/packages/babel-parser/src/parser/statement.ts#L1403-L1430